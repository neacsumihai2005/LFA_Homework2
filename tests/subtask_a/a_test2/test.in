3
3 4 5
9
3 3 a
3 4 b
4 4 a
4 5 a
4 5 b
5 4 a
5 4 b
3 5 b
5 3 a
3
1
5
