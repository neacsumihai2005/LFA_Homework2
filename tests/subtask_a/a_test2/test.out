3 -> q0
4 -> q1
5 -> q2

{q0}: ({q0}, a) ({q1, q2}, b) 
{q1, q2}: ({q0, q1, q2}, a) ({q1, q2}, b) 
{q0, q1, q2}: ({q0, q1, q2}, a) ({q1, q2}, b) 

